src/skins/tu.tsx

```diff
@@ -0,0 +1,315 @@
+            textButtonPrimaryInverse: palette.primary,
+            textButtonPrimaryInverseSelected: palette.primary,
+            textButtonSecondary: palette.primary,
+            textButtonSecondarySelected: palette.primary80,
+            textButtonSecondaryInverse: palette.white,
+            textButtonSecondaryInverseSelected: palette.white,
+            textLink: palette.blue,
+            textLinkInverse: palette.white,
+            textLinkDanger: palette.red,
+            textLinkSnackbar: palette.white,
+            control: palette.grey3,
+            controlActivated: palette.primary,
+            controlInverse: palette.white,
+            controlActivatedInverse: palette.white,
+            controlError: palette.red,
+            barTrack: palette.grey2,
+            loadingBar: palette.green40,
+            loadingBarBackground: palette.green,
+            toggleAndroidInactive: palette.grey2,
+            toggleAndroidBackgroundActive: palette.primary15,
+            iosControlKnob: palette.white,
+            divider: palette.grey2,
+            dividerInverse: applyAlpha(palette.white, 0.2),
+            navigationBarDivider: palette.blue,
+            badge: palette.red70,
+            feedbackErrorBackground: palette.red,
+            feedbackInfoBackground: palette.primary,
+            brand: palette.blue,
+            brandHigh: palette.blue70,
+            brandLow: palette.blue10,
+            inverse: palette.white,
+            neutralHigh: palette.grey9,
+            neutralMedium: palette.grey5,
+            neutralMediumInverse: palette.grey5,
+            neutralLow: palette.grey1,
+            neutralLowAlternative: palette.grey2,
+            textPrimary: palette.grey9,
+            textPrimaryInverse: palette.white,
+            textSecondary: palette.grey6,
+            textSecondaryInverse: palette.grey3,
+            error: palette.red,
+            success: palette.green,
+            warning: palette.orange,
+            promo: palette.blue,
+            highlight: palette.red40,
+            successLow: palette.green10,
+            warningLow: palette.orange20,
+            errorLow: palette.red10,
+            promoLow: palette.blue10,
+            successHigh: palette.green75,
+            warningHigh: palette.orange70,
+            errorHigh: palette.red70,
+            promoHigh: palette.blue,
+            successHighInverse: palette.green75,
+            warningHighInverse: palette.orange70,
+            errorHighInverse: palette.red70,
+            promoHighInverse: palette.blue,
+            textNavigationBarPrimary: palette.white,
+            textNavigationBarSecondary: palette.grey1,
+            textNavigationSearchBarHint: palette.grey1,
+            textNavigationSearchBarText: palette.white,
+            textAppBar: palette.grey4,
+            textAppBarSelected: palette.blue,
+            customTabsBackground: palette.blue,
+        },
+        darkModeColors: {
```

Comment:
the dark mode in the Tu brand is not very elaborate. We will improve it in the future